src/ts/stringutils.ts

```diff
@@ -112,7 +114,28 @@ export namespace StringUtils {
         time = player.getCurrentTime();
       } else if (formatString.indexOf('adDuration') > -1) {
         time = player.getDuration();
+      } else if (formatString.indexOf('adBreakRemainingTime') > -1) { // To display the remaining time in the ad bread as opposed to in the ad
+        time = 0;
+        
+        // compute list of ads and calculate duration of remaining ads based on index of active ad
+        if (player.ads.isLinearAdActive()) {
+          const isActiveAd = (ad : Ad) => player.ads.getActiveAd().id == ad.id;
+          const indexOfActiveAd = player.ads.getActiveAdBreak().ads.findIndex(isActiveAd)
+          const duration = player.ads.getActiveAdBreak().ads.slice(indexOfActiveAd)
+            .map(ad => {
+              if(ad.isLinear) {
+                return (ad as LinearAd).duration
+              } else {
+                return 0
+              }
+            })
+            .reduce((prev, current) => prev + current, 0);
```

Comment:
nit: this could be simplified to use a single `reduce`:

```suggestion
          const duration = player.ads.getActiveAdBreak().ads
	    .slice(indexOfActiveAd)
            .reduce((total, ad) => total + (ad.isLinear ? (ad as LinearAd).duration : 0), 0);
```